pkg/local_object_storage/blobstor/put.go

```diff
@@ -44,7 +44,7 @@ func (b *BlobStor) Put(prm common.PutPrm) (common.PutRes, error) {
 			res, err := b.storage[i].Storage.Put(prm)
 			if err == nil {
 				logOp(b.log, putOp, prm.Address, b.storage[i].Storage.Type(), res.StorageID)
-			} else if errors.Is(err, common.ErrNoSpace) {
+			} else if errors.Is(err, common.ErrNoSpace) && i < len(b.storage)-1 {
```

Comment:
I don't think you can do it this way. The next one can have a policy that doesn't match.

Reply:
nice catch. In theory, policies may be any with any order, so we should be ready